webapp/src/api/configuration/ConfigurationApi.tsx

```diff
@@ -0,0 +1,16 @@
+import { HttpMethod } from 'simple-http-request-builder';
+import ApiHttpClient from '../ApiHttpClient';
+import { Configuration } from '../types/MovieDbTypes';
+
+export default class ConfigurationApi {
+  private baseUrl: String = '/configuration';
+
+  constructor(private readonly httpClient: ApiHttpClient) {}
+
+  getConfiguration() {
+    return this
+      .httpClient
+      .restRequest<Configuration>(HttpMethod.GET, `${this.baseUrl}`)
```

Comment:
pas besoin de template string 
```suggestion
      .restRequest<Configuration>(HttpMethod.GET, this.baseUrl)
```